invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentDetails/components/DocumentPendingLoans/DocumentPendingLoans.js

```diff
@@ -30,7 +31,13 @@ export default class DocumentPendingLoans extends Component {
   _showAllHandler = () => {
     const { document_pid } = this.props.document.id;
     this.props.history.push(
-      this.showAllUrl(document_pid, null, 'state:PENDING', null)
+      this.showAllUrl(
+        loanApi
+          .query()
+          .withDocPid(document_pid)
+          .withState('PENDING')
```

Comment:
Referring to the options pattern above this could be written as:
```
        loanApi
          .query()
          .withParams({
              document_pid: document_pid,
              state: 'PENDING'
          })
          .build()
          ...

```